src/utils/constants.ts

```diff
@@ -268,8 +268,8 @@ export const DATA_TYPES: string[] = [
 ];
 
 export const NAME_TYPES: { [key: string]: string }[] = [
-  { value: "FULLY_SPECIFIED", label: "Fully Specified" },
   { value: "null", label: "Synonym" }, // value here should be null but inputs html can't handle null as a value
+  { value: "FULLY_SPECIFIED", label: "Fully Specified" },
```

Comment:
I'm less sure about this... "Fully Specified" should definitely be the first option we display when creating a new name, so maybe we just don't change this for now?

Reply:
Incidentally, I think this is why there's a test failure. The test assumes that the first name entered is always a FSN and for any concept in any language we always require at least one FSN

Reply:
Thanks, for catching this test failure, I will revert it.